bedrock/careers/templates/careers/listings.html

```diff
@@ -12,40 +12,37 @@
 {% block body_class %}listings-page{% endblock %}
 
 {% block careers_content %}
-<section class="mzp-c-hero mzp-t-dark page-hero">
-  <div class="mzp-l-content">
-    <div class="mzp-c-hero-body">
-      <h1 class="mzp-c-hero-title">Feel good about your work again</h1>
-
-      <div class="mzp-c-hero-desc">
-        <p>
-          Join us.
-        </p>
-      </div>
+<header class="c-careers-text-hero">
+  <div class="mzp-l-content c-text-hero-wrapper mzp-t-content-md">
+    <div class="c-text-hero-title">
+      <h1>Feel good about your work again</h1>
+    </div>
+    <div class="c-text-hero-body">
+      <p>Bring your drive, your creativity, your big ideas and your new perspectives to make the difference we’re aiming for.</p>
     </div>
   </div>
-</section>
+</header>
 
-<section class="mzp-l-content">
+<section class="mzp-l-content mzp-t-content-xl">
   <h2>Listings</h2>
```

Comment:
**Suggestion:** I would probably add a `margin-bottom` to this header to create a better degree of separation between title and listings

Reply:
yeah, that's one of the general tweaks I wanted to make... in styles.scss you set the top and bottom padding of `mzp-l-content` to 16px globally across all these pages, which then creates some places were spacing has to be added back that would otherwise already be part of the container. I'd like to make a pass through all the pages and try to make the spacing more consistent now that we can see how everything comes together.